src/neuroconv/datainterfaces/behavior/audio/requirements.txt

```
@@ -0,0 +1 @@
+ndx-sound==0.2.0
```

```suggestion
ndx-sound>=0.2.0
```

We've been getting more in the habit of lower bound pinning to avoid conflicts